TO-DO.md

```diff
@@ -0,0 +1,36 @@
+# General TODOs & Informal Notes
```

Comment:
The README should include TODO items to ensure that a single version of the document reflects both the current state of the project and the pending tasks.

Reply:
Meaning these should be kept in the README and we should remove the proposed TO-DO file?